core/client/fs/src/test/java/alluxio/client/block/policy/CapacityBaseRandomPolicyTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.block.policy;
+
+import alluxio.client.block.BlockWorkerInfo;
+import alluxio.client.block.policy.options.GetWorkerOptions;
+import alluxio.wire.WorkerNetAddress;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+public class CapacityBaseRandomPolicyTest {
+
+  @Test
+  public void getWorker() {
+    GetWorkerOptions getWorkerOptions = GetWorkerOptions.defaults();
+    ArrayList<BlockWorkerInfo> blockWorkerInfos = new ArrayList<>();
+    WorkerNetAddress netAddress1 = new WorkerNetAddress();
+    WorkerNetAddress netAddress2 = new WorkerNetAddress();
+    WorkerNetAddress netAddress3 = new WorkerNetAddress();
+    blockWorkerInfos.add(new BlockWorkerInfo(netAddress1, 10, 0));
+    blockWorkerInfos.add(new BlockWorkerInfo(netAddress2, 100, 0));
+    blockWorkerInfos.add(new BlockWorkerInfo(netAddress3, 1000, 0));
+    getWorkerOptions.setBlockWorkerInfos(blockWorkerInfos);
+    Assert.assertEquals(netAddress1, buildPolicyWithTarget(0).getWorker(getWorkerOptions));
+    Assert.assertEquals(netAddress1, buildPolicyWithTarget(7).getWorker(getWorkerOptions));
+    Assert.assertEquals(netAddress2, buildPolicyWithTarget(10).getWorker(getWorkerOptions));
+    Assert.assertEquals(netAddress2, buildPolicyWithTarget(70).getWorker(getWorkerOptions));
+    Assert.assertEquals(netAddress3, buildPolicyWithTarget(100).getWorker(getWorkerOptions));
+    Assert.assertEquals(netAddress3, buildPolicyWithTarget(700).getWorker(getWorkerOptions));
```

Comment:
Shall we add a couple of negative test cases?  e.g. your getworker might return null

Reply:
add one more test: testNoMatchWorker